source/runtime.cpp

```diff
@@ -1556,6 +1568,9 @@ bool reshade::runtime::load_effect(const std::filesystem::path &source_file, con
 		effect = {};
 		effect.source_file = source_file;
 		effect.source_hash = source_hash;
+
+		if (effect.source_file.extension() == L".addonfx")
+			effect.rendering = std::numeric_limits<unsigned int>::max();
```

Comment:
This effectively forces "*.addonfx" effects to always get uniform variables etc. updated, even if no technique inside has been enabled by the add-on (and thus it not actually rendering), which is not ideal.
I have attempted an alternative implementation in db5c3b8b31cb96c5079e4e0403f7e3780559a8d5, which should achieve the same goal, but without this drawback. I hope this solves this for you too?

Reply:
Capture is working. However, we can identify the extension at the effect is loaded, so it may be better to create a member that `bool is_addonfx`.